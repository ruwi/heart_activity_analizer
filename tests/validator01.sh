#!/bin/sh
python2 ../validator.py -f test.dat -l
echo '###'
cat test.dat | python2 ../validator.py -f test.dat -l
echo '###'
python2 ../validator.py -f test.dat -l --filter all
echo '###'
cat test.dat | python2 ../validator.py -f test.dat -l --filter all 
echo '###'
python2 ../validator.py -f test.dat -l --filter valid
echo '###'
cat test.dat | python2 ../validator.py -f test.dat -l --filter valid
echo '###'
python2 ../validator.py -f test.dat -l --filter invalid
echo '###'
cat test.dat | python2 ../validator.py -f test.dat -l --filter invalid
echo '###'
