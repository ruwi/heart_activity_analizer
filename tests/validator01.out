Lines of all data: 5
###
Lines of all data: 5
###
Lines of all data: 5
###
Lines of all data: 5
###
Lines of valid data: 2
###
Lines of valid data: 2
###
Lines of invalid data: 3
###
Lines of invalid data: 3
###
